lib/accessories/thermostat.js

```diff
@@ -35,9 +35,11 @@ function FritzThermostatAccessory(platform, ain) {
 
     // Thermostat
     this.services.Thermostat.getCharacteristic(Characteristic.CurrentHeatingCoolingState)
+        .setProps({validaValues: [0, 1]})
```

Comment:
validaValues looks like a typo- did you test this?

Reply:
This is indeed a typo. Gonna correct it now. It is indeed tested and fully working. However this validValue props for the `CurrentHeatingCoolingState` seems to have slipped as it hasn't  really an effect on the experience. It's just there for completeness.
The relevant part is the validValues array for the `TargetHeatingCoolingState` (where is no typo lol) which restricts the controls in the Home App to on and off.

Reply:
typo fixed ✅